### PR TITLE
CI: use the token provided from Github

### PR DESCRIPTION
**Pull request by naveen521kk**
raise an error when corresponding env vars aren't set
See https://github.com/saadpasta/developerFolio/pull/334#issuecomment-792272372

### CODE REVIEW

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: Build and Deploy
 env:
   CI: false
   GITHUB_USERNAME: ${{ github.repository_owner }}
-  REACT_APP_GITHUB_TOKEN: ${{ secrets.OPEN_SOURCE_TOKEN }} # Use the generated personal access token and add it to repository secrets with the name OPEN_SOURCE_TOKEN
+  REACT_APP_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # Use the generated personal access token and add it to repository secrets with the name OPEN_SOURCE_TOKEN
```
[the remark this answers is not in the document]
Comment (naveen521kk) [reply .github/workflows/node.js.yml:5]: I think it should be fine to do use `GITHUB_TOKEN` directly, it should have the necessary permissions for that.

[the remark this answers is not in the document]
Comment (saadpasta) [reply .github/workflows/node.js.yml:5]: okay

**Review by saadpasta — APPROVED** [review]

**Comment by naveen521kk** [comment]
It works :D https://developerfolio.js.org/#contact

**Comment by naveen521kk** [comment]
@saadpasta I think you should configure vercel so that it works without any error.

**Comment by saadpasta** [comment]
> @saadpasta I think you should configure vercel so that it works without any error.

Yeah, I try to that but still, it would need my authentication.

**Pull request merged**